lib/rmt/cli/base.rb

```diff
@@ -16,6 +16,10 @@ def help(shell, subcommand = false)
       class_options_help(shell)
 
       shell.say "Run '#{basename} help [COMMAND]' for more information on a command and its subcommands."
+
+      shell.say
+      shell.say 'Do you have suggestions for improvement? Let us know!'
+      shell.say 'Check out https://github.com/SUSE/rmt#feedback'
```

Comment:
@felixsch maybe we could link directly to a new issue? That would be -1 click

Reply:
thought about this as well so yes why not